<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20904" started-at="2020-07-24T01:50:35 EDT" finished-at="2020-07-24T01:50:56 EDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20904" started-at="2020-07-24T01:50:35 EDT" finished-at="2020-07-24T01:50:56 EDT">
      <class name="booking.LoginWithValidCredentialUsingEmail">
        <test-method status="PASS" signature="startTestSuite()[pri:0, instance:booking.LoginWithValidCredentialUsingEmail@7fac631b]" name="startTestSuite" is-config="true" duration-ms="24" started-at="2020-07-24T01:50:35 EDT" finished-at="2020-07-24T01:50:35 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTestSuite -->
        <test-method status="PASS" signature="beforeEachTestMethod()[pri:0, instance:booking.LoginWithValidCredentialUsingEmail@7fac631b]" name="beforeEachTestMethod" is-config="true" duration-ms="13905" started-at="2020-07-24T01:50:35 EDT" finished-at="2020-07-24T01:50:49 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeEachTestMethod -->
        <test-method status="FAIL" signature="loginWithValidCredentialUsingEmailTest()[pri:0, instance:booking.LoginWithValidCredentialUsingEmail@7fac631b]" name="loginWithValidCredentialUsingEmailTest" duration-ms="6965" started-at="2020-07-24T01:50:49 EDT" finished-at="2020-07-24T01:50:56 EDT">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [mohammad uddin Genius Level 1 ] but found [mohammad uddin
Genius Level 1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [mohammad uddin Genius Level 1 ] but found [mohammad uddin
Genius Level 1]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at booking.LoginWithValidCredentialUsingEmail.loginWithValidCredentialUsingEmailTest(LoginWithValidCredentialUsingEmail.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithValidCredentialUsingEmailTest -->
        <test-method status="PASS" signature="afterEachTestMethod()[pri:0, instance:booking.LoginWithValidCredentialUsingEmail@7fac631b]" name="afterEachTestMethod" is-config="true" duration-ms="0" started-at="2020-07-24T01:50:56 EDT" finished-at="2020-07-24T01:50:56 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterEachTestMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:booking.LoginWithValidCredentialUsingEmail@7fac631b]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-07-24T01:50:56 EDT" finished-at="2020-07-24T01:50:56 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- booking.LoginWithValidCredentialUsingEmail -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
